src/FormInterface.php

```diff
@@ -4,21 +4,241 @@
 
 namespace Yiisoft\Form;
 
-interface FormInterface
+use Yiisoft\Validator\DataSetInterface;
+
+interface FormInterface extends DataSetInterface
 {
-    public function addError(string $attribute, string $error): void;
+    /**
+     * Adds a list of errors.
+     *
+     * @param array $items a list of errors. The array keys must be attribute names.
+     * The array values should be error messages. If an attribute has multiple errors, these errors must be given in
+     * terms of an array.
+     *
+     * You may use the result of {@see errors()} as the value for this parameter.
+     */
     public function addErrors(array $items): void;
-    public function attributeLabels(): array;
+
+    /**
+     * Adds a new error to the specified attribute.
+     *
+     * @param string $attribute attribute name
+     * @param string $error new error message
+     */
+    public function addError(string $attribute, string $error): void;
+
+    /**
+     * Removes errors for all attributes or a single attribute.
+     *
+     * @param string|null $attribute attribute name. Use null to remove errors for all attributes.
+     */
     public function clearErrors(?string $attribute = null): void;
```

Comment:
What's the use case for the method?

Reply:
Allows you to clear errors before running the validator.

Reply:
But the `validate()` is part of the form so it could clear it without exposing a public method, can't it?

Reply:
Change visibility done.